CMakeLists.txt

```diff
@@ -103,7 +103,7 @@ else (WIN32)
       # Ensure that objcopy is present
       if (DEFINED ENV{CROSSCOMPILE} AND NOT DEFINED CLR_CROSS_COMPONENTS_BUILD)
         if (CMAKE_SYSTEM_PROCESSOR STREQUAL armv7l OR CMAKE_SYSTEM_PROCESSOR STREQUAL aarch64)
-          find_program(OBJCOPY ${TOOLCHAIN_PREFIX}objcopy)
+          find_program(OBJCOPY ${TOOLCHAIN}-objcopy)
```

Comment:
What about replace `${TOOLCHAIN_PREFIX}` to `${TOOLCHAIN}` and redefine other macro for the previous `${TOOLCHAIN}` variable? I can revise the find objcopy logic as you said. But I think this issue can be resolved by modifiying `cross/android/arm64/toolchain.cmake` file. What do you think?